### DISCUSSION TITLE
How to publish a traveling meeting on google sheets tsmlui?

### DISCUSSION THREAD

**Discussion by ghostname3** [code4recovery/12-step-meeting-list#1379] [Q&A] [answered]
This is a traveling meeting that never meets at the same destination. It meets at different times and days once a week. Meetings are announced via email Wednesday before. All info found here: https://sites.google.com/view/thenomadclub/home

I'm stumped. Hoping someone can come up with a helpful solution.

**Comment by joshreisner** [comment]
could you update the address on the listing every wednesday?

**Reply by ghostname3** [reply]
Thanks for the reply. The meeting itself isn't on Wednesday. The meeting announces the meeting on Wednesday for the following week. The announced meeting could be any day or location or time the following week.

**Reply by joshreisner** [reply]
ok, the time varies too! well if you can update the listing each week, but no more than 7 days ahead of time, that would be ideal. as long as the next occurrence of the meeting is always accurate, that should be fine

**Reply by ghostname3** [reply]
Is there a way to post the meeting without having to have a time and address? Or is there a way to post Mon-Sun without an address and point end user to the meeting's website for further info? There's too many variables for the sheets manager to stay on it weekly. There's too much room for error.

Is there precedence for this type of meeting on tsmlui to your knowledge? Thanks for your comms.

**Reply by joshreisner** [reply, marked as answer]
no, i'm afraid not. if the meeting can't stay updated, it shouldn't be posted to the directory. 

i have heard of traveling meetings, or meetings at a member's home where the address can't be shared as a matter of privacy. in these cases we recommend using another page on the site to mention them. perhaps a page called something like "Additional Recovery Resources"

**Reply by ghostname3** [reply]
Thank you very much for your feedback. Consider this answered.
